c/extras/aggregator.cc

```diff
@@ -282,10 +285,12 @@ void Aggregator::group_2d_continuous(DataTablePtr& dt,
 
   #pragma omp parallel for schedule(static)
   for (int64_t i = 0; i < dt->nrows; ++i) {
-    double v0 = ISNA<double>(d_c0[i])? 0 : d_c0[i];
-    double v1 = ISNA<double>(d_c1[i])? 0 : d_c1[i];
-    d_groups[i] = static_cast<int32_t>(normy_factor * v1 + normy_shift) * nx_bins +
-                  static_cast<int32_t>(normx_factor * v0 + normx_shift);
+    if (ISNA<double>(d_c0[i]) || ISNA<double>(d_c1[i])) {
+      d_members[i] = NA_I4;
```

Comment:
`GETNA<int32_t>()` is a more C++-ish way. Although more verbose, at least these constants are strictly typed.

Reply:
Fixed.